packages/test/test-service-load/src/runner.ts

```diff
@@ -176,7 +176,7 @@ async function runnerProcess(
 	// Assigning no-op value due to linter.
 	let metricsCleanup: () => void = () => {};
 
-	const optionsOverride = `${driver}${endpoint !== undefined ? `-${endpoint}` : ""}`;
+	const optionsOverride = endpoint ?? driver;
 	const loaderOptions = generateLoaderOptions(
 		seed,
 		runConfig.testConfig?.optionOverrides?.[optionsOverride]?.loader,
```

Comment:
there is some discussion here that might help understand driver vs endpoint: https://github.com/microsoft/FluidFramework/pull/14984#issuecomment-1500478047

Reply:
If I understand this suggestion correctly, for a particular driver say "odsp", you can add configs and options in testConfig.json in 2 ways:
1. Under "odsp". This would apply to all odsp driver types.
2. You can override configs / options provided above for a specific endpoint by adding them under say "odsp-df" or "odsp-local".

Reply:
Also, do you know why the endPointName for "odsp" driver is "odsp" by default instead of empty string? Specifying configs / options for "odsp-odsp" seems weird.
Same for "tinylicious". The endPointName is "local" which is so weird.

Reply:
for odsp, the endpoints should probably be dogfood, and prod. i'm fine with local for local driver and tinylicious, but i don't really understand why they need one at all. its an area that needs clean up.

my suggestion for the config change above, would allow configs under driver type (odsp) which can be overridden with type+endpoint (odsp-odsp, or odsp-odsp-df)

Reply:
In all honestly, I would prefer my original solution until we come to a consensus on what the optionOverride should be based on. Yes, `odsp-odsp` and `odsp-odsp-df` aren't my favorite names, but at least we have consistency. It's probably better to keep everything the same, and then when we actually come to a consensus, we can update everything at once, and it would be easier to find, and less confusing across our telemetry. No one will misinterpret `odsp-odsp` or `odsp-odsp-df`

Reply:
Updated it. To not include any unknown side effects, I've forced the config option to only take what is seen. If we want duplicate, I feel like it would require more changes to how we do the test configs. I think what we have now is fine.

Reply:
looks good. Please update the PR description based on this change :-).